src/tracer.js

```diff
@@ -239,7 +241,11 @@ export default class Tracer {
         ctx.baggage = parent.baggage;
       }
 
-      ctx.traceId = randomId;
+      if (options.traceId128bit) {
+        ctx.traceId = new Buffer.concat([Utils.getRandom64(), randomId]);
```

Comment:
I believe you can just drop the `new` to avoid the deprecated constructor

Reply:
In this case I think the use of `new` is unnecessary, regardless of the constructor being deprecated.